samples/templates/default-azuredeploy-docker.json

```diff
@@ -404,20 +405,35 @@
             ]
         },
         {
-            "apiVersion": "2015-05-01",
-            "name": "[variables('appInsightsName')]",
+            "type": "Microsoft.OperationalInsights/workspaces",
+            "apiVersion": "2022-10-01",
+            "name": "[variables('appInsightsWorkspaceName')]",
+            "location": "[parameters('applicationInsightsLocation')]",
+            "condition": "[variables('deployAppInsights')]",
+            "tags": {
+                "Privacy.Asset.NonPersonal": ""
+            }
+        },
+        {
             "type": "Microsoft.Insights/components",
+            "apiVersion": "2020-02-02",
+            "name": "[variables('appInsightsName')]",
             "location": "[parameters('applicationInsightsLocation')]",
             "condition": "[variables('deployAppInsights')]",
+            "dependsOn": [
+                "[resourceId('Microsoft.OperationalInsights/workspaces', variables('appInsightsWorkspaceName'))]"
+            ],
             "kind": "web",
             "tags": {
                 "[concat('hidden-link:', variables('appServiceResourceId'))]": "Resource",
                 "displayName": "AppInsightsComponent",
-                "FhirServerSolution": "[parameters('solutionType')]"
+                "FhirServerSolution": "[parameters('solutionType')]",
+                "Privacy.Asset.NonPersonal": ""
```

Comment:
is this needed in OSS?

Reply:
When this script is executed, it creates App Insights resources that can trigger S360 alerts in our subscriptions. 
To minimize the number of S360 alerts I thought about adding this tag during the creation of these App Insights.

Reply:
I don't think this tag is correct for all OSS deployments. We should check w/ Joyce, but my understanding is If you deploy a test OSS solution and you login with your IP address, upn, etc that is still personal data even if it's only employee data.

I would recommend exposing an "additionalTags" variable that is an object and combining it with this tag object. I've seen this done in the past